### PR TITLE
Adds the ability to hide tiny items in vents

### PR DESCRIPTION
**Pull request by Amelia0010**
[This, but it works](https://github.com/HippieStation/HippieStation/pull/7960)

#### Changelog

:cl:  YoYoBatty
rscadd: Adds ability to crowbar and hide tiny items in vent pumps
/:cl:

### CODE REVIEW

**Comment by Nickvr628** [comment]
So the captain can hide the nuke disk in a vent under a table under an RCD'ed wall.

Nice

**Comment by yoyobatty** [comment]
Except not really because vents by default appear ontop of walls.

**Comment by yoyobatty** [comment]
Alright, update the code to sync with the original pr and it should be good to go.

File: yogstation/code/modules/atmospherics/unary_devices/vent_pump.dm [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+/obj/machinery/atmospherics/components/unary/vent_pump/crowbar_act(mob/living/user, obj/item/I)
+	to_chat(user, "<span class='notice'>You begin prying [cover ? "in" : "off"] the vent cover.</span>")
+	if(I.use_tool(src, user, 50, volume=50))
+		cover = !cover
+		to_chat(user, "<span class='notice'>You pry [cover ? "off" : "in"] the vent cover.</span>")
+	return TRUE
+
+/obj/machinery/atmospherics/components/unary/vent_pump/attackby(obj/item/W, mob/user, params)
+	if(cover && !welded)
+		if(W.w_class == WEIGHT_CLASS_TINY && istype(W,/obj/item) && user.a_intent != INTENT_HARM)
```
Comment (ThatLing) [inline yogstation/code/modules/atmospherics/unary_devices/vent_pump.dm]: Switch the class and istype checks around. If W is always ``/obj/item``, you don't need the typecheck. If it isn't, ``w_class`` is defined in ``/obj/item`` and you only check if it's the correct type after accessing ``w_class``.

**Review by ThatLing — CHANGES_REQUESTED** [review]

**Pull request merged**